### PR TITLE
Implement global templates container and filtering by templates on alloc...

### PR DESCRIPTION
**Pull request by deemoowoor**
...ate (OMS-253)

### CODE REVIEW

File: opennode/knot/model/compute.py [hunk 1]
```diff
@@ -201,6 +201,8 @@ class Compute(Container):
     __markers__ = [IVirtualCompute, IDeployed, IUndeployed, IDeploying, IZabbixConfiguration, IManageable,
                    ISaltInstalled, IFuncInstalled]
 
+    hostname = u''
```
Comment (livenson) [inline opennode/knot/model/compute.py:204]: Why was this needed?

Comment (deemoowoor) [inline opennode/knot/model/compute.py:204]: I was getting exceptions about Compute not having the hostname attribute. Must be a database corruption or something. Anyway, let's leave it for now. I'll create a new issue about it.

Comment (deemoowoor) [inline opennode/knot/model/compute.py:204]: Issue number is OMS-260
